### ISSUE TITLE
Feature request: More configuration options

### ISSUE THREAD

**Issue by alexander-svendsen** [nutgaard/yet-another-fetch-mock#2]
It is possible to configure this ourselves through the middleware api, but it would be more epic if the library had native support for these things:

```js
const mock = FetchMock.configure({
  enableFallback: boolean,
  delay: number,         // each request is delayed by ms
  failureRate: number // unsure about this one, mabye % of request that fails
  
});

````

**Comment by alexander-svendsen** [comment]
Hmm maybe add some labels to this project, so we can mark the things more correctly ?

**Comment by nutgaard** [comment]
I like keeping the number of options as low as possible. 
But adding helper-functions to compose middlewares could be done. 

E.g 
```
const mock = FetchMock.configure({
  enableFallback: boolean,
  middleware: combine(
    delayMiddleware(1000),
    failurereateMiddleware(0.1),
    loggingMiddleware
  )
});
```

Or
```
const mock = FetchMock.configure({
  enableFallback: boolean,
  middleware: [
    delayMiddleware(1000),
    failurereateMiddleware(0.1),
    loggingMiddleware
  ]
});
```

**Comment by nutgaard** [comment]
Solved by #3 
Solution ended up being: 
```
const delayedErrorMock = FetchMock.configure({
  middleware: MiddlewareUtils.combine(
    MiddlewareUtils.delayMiddleware(200),
    MiddlewareUtils.failurerateMiddleware(0.2)
  )
});
```

Ofc the `MiddlewareUtils.` prefix can be removed using `import {combine, delayMiddleware, failurerateMiddleware} from '...'`
